src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/OutlineTopComponent.java

```diff
@@ -205,6 +217,19 @@ protected boolean requestFocusInWindow(boolean temporary) {
 
     @Override
     public void resultChanged(LookupEvent lookupEvent) {
+        // Highlight the focused graph, if available, in the outline.
+        if (result.allItems().isEmpty()) {
```

Comment:
It looks like you are confident that result is not NULL when resultChanged() is called.  I believe framework calls it after `componentOpened()`.  if so, is it possible to remove result = null in `componentClosed()`?

Reply:
Right, see e.g. the code listed in https://netbeans.apache.org/tutorials/nbm-selection-1.html#_creating_a_context_sensitive_topcomponent which follows the same pattern.

`result = null` is just added for consistency with other TopComponent classes in the project, e.g:

https://github.com/openjdk/jdk/blob/b2a58bec4a4f70a06b23013cc4c351b36a413521/src/utils/IdealGraphVisualizer/ControlFlow/src/main/java/com/sun/hotspot/igv/controlflow/ControlFlowTopComponent.java#L134

But you are right, it does not seem necessary and I just removed it.